src/wp-includes/http.php

```diff
@@ -365,25 +366,7 @@ function wp_remote_retrieve_cookie_value( $response, $name ) {
  * @return bool
  */
 function wp_http_supports( $capabilities = array(), $url = null ) {
```

Comment:
Since this is `@deprecated` now, we should call `_deprecated_function( __FUNCTION__, '1.0.0-beta1' );` here too.

Reply:
Yes and no - if we do that now we generate a lot of noise in plugins that are unlikely to add suitable guards.

Reply:
OK, let's go with this for now then.